presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -186,9 +207,21 @@ public String getQuery()
     }
 
     @JsonProperty
-    public Duration getElapsedTime()
+    public long getExecutionTimeMillis()
     {
-        return elapsedTime;
+        return executionTime.toMillis();
+    }
+
+    @JsonProperty
+    public long getCpuTimeMillis()
+    {
+        return cpuTime.toMillis();
+    }
+
+    @JsonProperty
+    public long getElapsedTimeMillis()
```

Comment:
This doesn't look like it's going to round trip anymore. On the other hand, I'm not sure we ever deserialize these. Either way though, either remove the JSON constructor or fix these so that it round-trips